browser/brave_wallet/brave_wallet_provider_impl_unittest.cc

```diff
@@ -436,8 +441,8 @@ class BraveWalletProviderImplUnitTest : public testing::Test {
 
     base::RunLoop run_loop;
     provider()->SignTypedMessage(
-        address, message, message_to_sign, std::move(domain), domain_hash,
-        primary_hash,
+        address, message, std::move(domain_hash), std::move(primary_hash),
```

Comment:
no need to move, it takes const reference and we don't usually move const reference for another const reference

Reply:
done